packages/jest-environment-puppeteer/README.md

```diff
@@ -60,6 +60,22 @@ it('should fill an input', async () => {
 })
 ```
 
+### `global.context`
+
+**Note: Requires puppeteer >= 1.5.0**
```

Comment:
Does it worth mentioning that? What about updating the peerDependancy: https://github.com/smooth-code/jest-puppeteer/blob/master/packages/jest-environment-puppeteer/package.json#L24